Create ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md (#2700)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1021-15-0-enhance-dashboard-template-assignment-for-untyped-devices-graft-release-cd-7736.md

---
date: ""

> title: Enhance Dashboard Template Assignment for Untyped Devices  [GRAFT][release/cd] (#7736)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Enhance dashboard template assignment for untyped devices
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61669
version: 1021.15.0
---

> Enhance Dashboard Template Assignment for Untyped Devices  [GRAFT][release/cd] (#7736)

Reviewer: ```suggestion
Previously, when device had no type property assigned, Dashboard template section in dashboard settings was hidden. Now, in this situation, section will be shown but user will be informed that enabling dashboard template will be possible only when type is added to device. To make it easier, there is a button that will navigate user to Device management app device view.
Changes also includes fix for child devices issue- before new approach to dashboard templates has been implemented, it was possible to add type dashboard to child device that had no c8y_IsDevice property. It is now possible again and c8y_IsDevice property is not necessary.
```

> Enhance Dashboard Template Assignment for Untyped Devices  [GRAFT][release/cd] (#7736)

Author: The Claude AI suggested to use the following body:
```suggestion
Previously, dashboard templates could not be assigned to devices without a device type, which limited the ability to monitor these devices effectively. With this change, it is now possible to assign dashboard templates to devices that have no device type specified. This allows users to create dashboards for any device, regardless of whether it has a type assigned, improving the flexibility and coverage of device monitoring.
```

---
date: ""

> title: Enhance dashboard template assignment for untyped devices

Reviewer 2: ```suggestion
title: Dashboard template now visible for devices without type property
```

Reviewer: that would suggest that existing dashboard instance with assgined type (dashboard template) is now visible for devices without type property. That's not true- main change of this PR is that assigning type to dashboard (making them dashboard template) is now enhanced for devices without property

Reviewer 2: @jdre-c8y Please suggest a correct user-friendly phrasing

Reviewer: @BeateRixen actually this one small change in your suggestion should do the work
```suggestion
title: Dashboard template setting now visible for devices without type property
```

> title: Enhance dashboard template assignment for untyped devices

Reviewer: actually this one small change in your suggestion should do the work
```suggestion
title: Dashboard template setting now visible for devices without type property
```

product_area: Application enablement & solutions
change_type:
  - value: change-QHu1GdukP
    label: Feature
component:
  - value: component-YdSEScrEC
    label: Cockpit
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-61669
version: 1021.15.0
---

> Previously, when device had no type property assigned, Dashboard template section in dashboard settings was hidden. Now, in this situation, section will be shown but user will be informed that enabling dashboard template will be possible only when type is added to device. To make it easier, there is a button that will navigate user to Device management app device view.

Reviewer 2: ```suggestion
Previously, when a device had no type property assigned, the dashboard template section in the dashboard settings was hidden. Now, in this case, the section is visible but information is provided that the dashboard template is only enabled when a type is added to the device. Moreover a button is provided which navigates the user to the device view in the Device Management application.
```

> Changes also includes fix for child devices issue- before new approach to dashboard templates has been implemented, it was possible to add type dashboard to child device that had no c8y_IsDevice property. It is now possible again and c8y_IsDevice property is not necessary.

Reviewer 2: I think we have an extra ticket/entry for this change?

Reviewer: for the fix? we do https://cumulocity.atlassian.net/browse/MTM-61830

Reviewer 2: @jdre-c8y Then we should remove the last section from here.

> Changes also includes fix for child devices issue- before new approach to dashboard templates has been implemented, it was possible to add type dashboard to child device that had no c8y_IsDevice property. It is now possible again and c8y_IsDevice property is not necessary.

Reviewer 2: ```suggestion
```
